pooltool/physics/resolve/resolver.py

```diff
@@ -109,7 +109,7 @@ def default(cls) -> ResolverConfig:
             ball_pocket=BallPocketModel.CANONICAL,
             ball_pocket_params={},
             stick_ball=StickBallModel.INSTANTANEOUS_POINT,
-            stick_ball_params={"english_throttle": 0.5, "squirt_throttle": 1.0},
+            stick_ball_params={"english_throttle": 1.0, "squirt_throttle": 1.0},
```

Comment:
Do the changes to the model affect the amount of spin that off center hits create?

Either way, good call to set this to 1. This has proven confusing.

Reply:
This PR shouldn't have any impact on the speed to spin ratio except for this config change.  The spin should be directly proportional to speed from `w_B = v / I_m * vec`.  That's true even if the tip offset is scaled from `english_throttle`.  Both speed and spin should be more accurately modeled now, though.

This change is mostly in preparation for the changes accounting for tip geometry, and to avoid confusion.  I think that not accounting for tip geometry is the main reason that the spin doesn't feel right because the animation of the cue isn't matching the contact point.

Reply:
> I think that not accounting for tip geometry is the main reason that the spin doesn't feel right because the animation of the cue isn't matching the contact point.

💡 Oh this just clicked for me! Thanks for explaining that.